### DISCUSSION TITLE
Mediatek dimensity 1100

### DISCUSSION THREAD

**Discussion by Samiuwu123** [skyline-emu/skyline#1574] [Q&A]
the game does not open when using the dimensity 1100 processor on the poco x3 gt

**Comment by yhak22** [comment]
what game ?

**Reply by Samiuwu123** [reply]
everybody.

**Reply by yhak22** [reply]
well i think ur device gpu or cpu 
doesn't support skyline yet cause mali is pain for all emulators devs

**Reply by ihsbleh13** [reply]
> everybody.

Try kamiko it runs well on me with same SoC
